teamsAPI/v1/index.js

```diff
@@ -3,32 +3,50 @@ var BASE_API_PATH="/api/v1";
 
 module.exports = teamsAPI;
 
-teamsAPI.register = function(app, teams){
+teamsAPI.register = function(app, teamDB){
 
-    // GET - /teams 
+// GET - /teams 
 app.get(BASE_API_PATH + "/teams", (req,res)=>{
     console.log(Date()+" - GET /teams");
-    res.statusCode = 200;
-    res.send(teams);
-    
+    teamDB.find({},(err,teams)=>{
+        if(err){
+            console.log(Date()+" - "+err);
+            res.sendStatus(500);
+        }else{
+            res.statusCode=200;
+            res.send(teams);
```

Comment:
Please, change the code to remove the _id generated by nedb from the information that is sent to the user.

Reply:
changes done.